sample-data/src/main/java/cz/muni/fi/pa165/sample/SampleDataLoader.java

```diff
@@ -53,11 +57,18 @@ private void createMember() {
         member.setEmail("email@email.com");
         member.setGivenName("peter");
         member.setSurname("cibula");
-        member.setRegistrationDate(new Date());
 
         MemberRegisterDTO memberRegister = new MemberRegisterDTO();
         memberRegister.setMember(member);
-        memberRegister.setPassword("12345");
-        memberFacade.registerMember(memberRegister);
+        memberRegister.setPassword("12345");       
+        Long memberId = memberFacade.registerMember(memberRegister);
+        createLoan(memberId);
+    }
+    
+    private void createLoan(Long memberId){
+        CreateLoanDTO loan = new CreateLoanDTO();
+        loan.setBookId(1L);
```

Comment:
It might not be 1L, don't rely on DB implementation details. Pass the id as argument.